assets/embauche.html

```diff
@@ -97,9 +97,9 @@
 	<section id="result" class="js-only">
 		<p>
 			Cela coûtera
-			<span data-source="salsuperbrut" data-round title="Salaire super-brut">…</span>&nbsp;€
+			<span data-source="salaire_super_brut" data-round title="Salaire super-brut">…</span>&nbsp;€
 			par <span data-source="period">mois</span>
-			à mon <span class="employer_type">entreprise</span>.
+			à mon <span class="employer_type">entreprise</span><span id="salaire_super_brut_container" hidden>, ou <span data-source="cout_du_travail" data-round title="Coût du travail"></span>&nbsp;€ après déduction des aides différées</span>.
```

Comment:
It's actually not `salaire_super_brut_container` but `cout_du_travail_container` :confounded: